spec/shared_context.rb

```diff
@@ -121,3 +116,30 @@ def integration_context_debugging
     end
   end
 end
+
+def system_context
```

Comment:
@tmiotto I have a question about something you mentioned:

Also, you left some of the shared client load and mock hanging on before (each:). Please bring it here before splitting the method.

I've removed the clients from before(:each), but is it really necessary to remove the mocks? For outside, the same would not work. Maybe I did not quite understand what you said.